windows/autobuild/autobuild_windowsinstaller_1_prepare.ps1

```diff
@@ -1,7 +1,22 @@
 # Sets up the environment for autobuild on Windows
 
-# Get the source path via parameter
-param ([string] $sourcepath)
+
```

Comment:
Why do you need so many empty lines here?

Reply:
legacy. At one point, all scipts had parameters in line 1-20 and instructions in line 21-x
Adding Parameters/Procedure headers instead

Reply:
changed.